airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/spec.json

```diff
@@ -0,0 +1,41 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/google-analytics-v4",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Google Analytics V4 Spec",
+    "type": "object",
+    "required": ["credentials_json", "view_id", "start_date"],
+    "additionalProperties": false,
+    "properties": {
+      "credentials_json": {
+        "type": "string",
+        "title": "Credentials JSON",
+        "description": "The contents of the JSON service account key. Check out the <a href=\"https://docs.airbyte.io/integrations/sources/googleanalytics\">docs</a> if you need help generating this key.",
+        "airbyte_secret": true
+      },
+      "view_id": {
+        "type": "string",
+        "title": "View ID",
+        "description": "The ID for the Google Analytics View you want to fetch data from. This can be found from the <a href=\"https://ga-dev-tools.appspot.com/account-explorer/\">Google Analytics Account Explorer</a>.",
+        "airbyte_secret": true
+      },
+      "start_date": {
+        "type": "string",
+        "title": "Start Date",
+        "description": "A date in the format YYYY-MM-DD.",
+        "examples": ["2020-06-01"]
+      },
+      "window_in_days": {
+        "type": "integer",
+        "description": "The amount of days for each data-chunk begining from start_date. Bigger the value - faster the fetch. (Min=1, as for a Day; Max=364, as for a Year).",
```

Comment:
Yes, for other connectors that make sense. But for GAv4 - no, because google didn't limit requests by date range. For page limits we have pagination.